charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -129,106 +126,90 @@ spec:
         volumeMounts:
           - mountPath: /config
             name: server-config
-{{- end}}
-{{- if .Values.backupRestore.medusa.enabled }}
+      {{- end }}
+      {{- if (eq .Values.cassandra.version "4.0.0") }}
```

Comment:
Should this be ``if hasPrefix "4.0" .Values.cassandra.version`` (or the other way around, can't remember hasPrefix's order)

Reply:
Let me know if you decide to do that; I'll match it in my Stargate changes for 4.0 support.

Reply:
Changes pushed

Reply:
@jakerobb I updated the code to use `hasPrefix`.